requirements.txt

```diff
@@ -0,0 +1 @@
+compas @ git+https://github.com/compas-dev/compas@main
```

Comment:
If I remember right, Pypi don't accept git requirements.. maybe pin to beta?

Reply:
yep, this is only temporary, before release, we need to fix this

Reply:
https://github.com/compas-dev/compas_robots/issues/3